src/patternfly/_charts.scss

```diff
@@ -0,0 +1,50 @@
+// Chart colors
+
+// blue
+$pf-chart-color-blue-100: #8bc1f7 !default;
+$pf-chart-color-blue-200: #519de9 !default;
+$pf-chart-color-blue-300: #06c !default;
+$pf-chart-color-blue-400: #004b95 !default;
+$pf-chart-color-blue-500: #002f5d !default;
+
+// green
+$pf-chart-color-green-100: #bde2b9 !default;
+$pf-chart-color-green-200: #7cc674 !default;
+$pf-chart-color-green-300: #4cb140 !default;
+$pf-chart-color-green-400: #38812f !default;
+$pf-chart-color-green-500: #23511e !default;
+
+// cyan
+$pf-chart-color-cyan-100: #a2d9d9 !default;
+$pf-chart-color-cyan-200: #73c5c5 !default;
+$pf-chart-color-cyan-300: #009596 !default;
+$pf-chart-color-cyan-400: #005f60 !default;
+$pf-chart-color-cyan-500: #003737 !default;
+
+// purple
+$pf-chart-color-purple-100: #b2b0ea !default;
+$pf-chart-color-purple-200: #8481dd !default;
+$pf-chart-color-purple-300: #5752d1 !default;
+$pf-chart-color-purple-400: #3c3d99 !default;
+$pf-chart-color-purple-500: #2a265f !default;
+
+// yellow
+$pf-chart-color-yellow-100: #f9e0a2 !default;
```

Comment:
What do you think about naming these "gold" to match our core styles? I don't see why we should name them differently. Same applies for "grey" (black).

Also for these colors that match the core colors, how about we just import the `sass-utilities/colors.scss` file and reference the core variable names? That would be presuming that we want the colors to match if we update the colors in core. I think that's a safe bet. Then if we make updates in core that we don't want reflected in the charts, we can just remove the reference to the core var and put in the hex color instead.

Also can you rename this to `charts.scss`? The `_` denotes that the file is a [scss partial](https://sass-lang.com/guide#topic-4) to be included from another scss file.

Reply:
> What do you think about naming these "gold" to match our core styles? I don't see why we should name them differently. Same applies for "grey" (black).

what do you think about that? @gdoyle1

Reply:
@christiemolloy that's good with me!